### ISSUE TITLE
[1.12.2 Suggestion] Blacklist Items From The Transmutation Table

### ISSUE THREAD

**Issue by vizthex123** [sinkillerj/ProjectE#2046]
Title. It'd be useful for custom pack makers (like myself)

I can't find a config option or command for it either.

If it already exists (or there's an add-on that does this), please let me know how to add things to the blacklist.

**Comment by HNGamingUK** [comment]
From what I am aware there isn't a blacklist per say. 

However you can remove EMC values for a specific items with `/projecte removeEMC [ItemName]`

After which you can run `/projecte reloadEMC`

**Comment by vizthex123** [comment]
Yeah I know (I've done that multiple times) but I want some items to be processed in an Anti-Matter Relay, but not be transmutable.
